packages/webapp-libs/webapp-core/tsconfig.json

```diff
@@ -7,10 +7,12 @@
     "noImplicitOverride": true,
     "noPropertyAccessFromIndexSignature": true,
     "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true
+    "types": ["@testing-library/jest-dom"],
+    "noFallthroughCasesInSwitch": true,
+    "jsx": "react-jsx"
   },
   "files": [],
-  "include": [],
+  "include": ["./src/**/**.*"],
```

Comment:
Are you sure about this change? We have `tsconfig.lib.json` config where there is:
```"include": ["src/**/*.ts", "src/**/*.tsx"],```

Reply:
Indeed. I had some issues earlier without it, but it seems like it was a problem with my LSP, after restart it works just fine. This line is not needed.